libcudacxx/include/cuda/std/detail/libcxx/include/__concepts/invocable.h

```diff
@@ -49,8 +49,9 @@ concept __invoke_constructible = requires(_Fun&& __fun, _Args&&... __args) {
 template<class _Fn, class... _Args>
 _LIBCUDACXX_CONCEPT_FRAGMENT(
   _Invocable_,
-  requires(_Fn&& __fn, _Args&&... __args) //
-  (_CUDA_VSTD::invoke(_CUDA_VSTD::forward<_Fn>(__fn), _CUDA_VSTD::forward<_Args>(__args)...)));
+  requires(_Fn&& __fn, _Args&&... __args)(
```

Comment:
Note: this change was required as our macro machinery would otherwise try to create a token with a `::` inside. Wrapping it in braces helps there.

I am not sure how it worked before, but yeah macros